### ISSUE TITLE
Error Handling

### ISSUE THREAD

**Issue by spapadiamantis** [MontpellierRessourcesImagerie/microscope-metrics#19]
Microscope-metrics does not currently raise error exceptions. At the moment the module solely logs activity. Implementing error handling also entails identifying possible errors and their meaning to users.

An example of a possible error in this project  is a saturation error when the saturation present on an image does not permit deducting viable metrics.

**Comment by juliomateoslangerak** [comment]
Yes, We need to implement metrics specific errors that can be raised. 
For the moment we are thinking about 
- SaturationError

Any other kind of error that might be worth implementing because it is a common error and is not already implemented in the standard library?

Note we can 'raise' warnings with 
```python
import warnings
warnings.warn("Warning...........Message")
```

**Comment by juliomateoslangerak** [comment]
Concerning the saturation, it might be worth implementing an utility function ```check_saturation(image)``` that will raise this exception when image is saturated.
